machine/delta/delta_ag5648/efivar/series

```diff
@@ -0,0 +1 @@
+XX-Makefile-changes-for-ONIE_DNI_V1.patch
```

Comment:
What is this patch for?  Why is it needed?  What does it do?  No other machine needs this.